composer.json

```diff
@@ -42,12 +42,12 @@
         "silverstripe/webauthn-authenticator": "Adds a method to authenticate with security keys or built-in platform authenticators."
     },
     "extra": {
-        "branch-alias": {
-            "dev-master": "4.x-dev"
-        },
         "expose": [
             "client/dist"
-        ]
+        ],
+        "branch-alias": {
+            "dev-master": "5.x-dev"
+        }
```

Comment:
Can you put this back where it was? The reason we have it there is that you can delete it from patch branches without touching the `expose` config lines (in terms of git history). I know it's minor, but the only change required here is `4` -> `5`